src/lib/Scenes/Map/types.ts

```diff
@@ -12,3 +14,5 @@ export interface Tab {
   id: string
   text: string
 }
+
+export type Show = GlobalMap_viewer["city"]["shows"]["edges"][0]["node"]
```

Comment:
Any chance this could be null?

Reply:
It could be null if we change the graphql fragment `GlobalMap_viewer` but that happens typescript will complain anyway so it's safe